### PR TITLE
MINOR: Clarify wording

### PR DESCRIPTION
**Pull request by jeffwidman**

### CODE REVIEW

**Comment by jeffwidman** [comment]
@ijuma Can you take a look at this one too? Should be straightforward

File: config/server.properties [hunk 1]
```diff
@@ -103,8 +103,8 @@ offsets.topic.replication.factor=1
 # The minimum age of a log file to be eligible for deletion due to age
 log.retention.hours=168
 
-# A size-based retention policy for logs. Segments are pruned from the log as long as the remaining
-# segments don't drop below log.retention.bytes. Functions independently of log.retention.hours.
+# A size-based retention policy for logs. Segments are pruned from the log until the remaining
+# segments drop below log.retention.bytes. Functions independently of log.retention.hours.
```
Comment (ijuma) [inline config/server.properties]: It seems like there's a slight change in meaning. Previously we said that we will keep at least log.retention.bytes where the new wording says that we will go below log.retention.bytes.

Comment (jeffwidman) [reply config/server.properties]: Agreed. Updated the PR to use `unless`.

**Review by ijuma — COMMENTED** [review]
LGTM, merging to trunk.

**Pull request closed without merging**